etc/containerbuddy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "consul": "{{ .CONSUL }}:8500",
+  "onStart": "/usr/local/bin/manage.sh start",
+  "services": [
+    {
+      "name": "couchbase",
+      "port": 80,
```

Comment:
This will be a couple services, one for each of the ports we'll need to care about.